components/block/faq.vue

```diff
@@ -1,76 +1,117 @@
 <template>
-  <div class="bg-neutral-300">
+  <div class="bg-gray-50">
     <div class="max-w-7xl mx-auto py-12 px-4 sm:py-16 sm:px-6 lg:px-8">
-      <div class="max-w-3xl mx-auto divide-y-2 divide-grey-200">
-        <h2
-          class="text-center text-3xl font-extrabold text-grey-800 sm:text-4xl"
-        >
-          Frequently asked questions
+      <div class="max-w-3xl mx-auto divide-y divide-[#C4C4C4]">
```

Comment:
you have colors black-500, grey-300, neutral-500 take look on design in a black frame